lib/sidekiq/symbols.rb

```diff
@@ -2,31 +2,44 @@
 
 module Sidekiq
 module Symbols
+  def self.symbolize_args(args)
+    symbolize_keys = lambda do |arg|
+      h = {}
+      arg.each do |k, v|
+        k = k.to_sym if k.respond_to?(:to_sym)
+        h[k] = v.is_a?(Hash) ? symbolize_keys.call(v) : v
+      end
+      h
+    end
+
+    symbolized_args = args.map do |arg|
+      if arg.is_a?(Hash)
+        symbolize_keys.call(arg)
+      else
+        arg
+      end
+    end.to_a
+  end
+
   def self.included(klass)
     klass.class_eval { prepend(Symbolizer) }
+
+    klass.class_eval do
+      def self.inherited(subclass)
```

Comment:
That's a great point. Last one wins. If `include Sidekiq::Symbols` came before the `def self.inherited` in the class, then the definition in the class would take precedence.

```ruby
module M
  def self.included(klass)
    klass.class_eval do
      def self.inherited(subclass)
        puts "inherited defined by M inclusion"
      end
    end
  end
end

class Base
  def self.inherited(subclass)
    p "inherited from within Base"
  end

  include M
end

class X < Base; end
# => inherited defined by M inclusion
```

Compared to:

```ruby
class Base
  include M

  def self.inherited(subclass)
    puts "inherited from within Base"
  end
end

class X < Base; end
# => inherited from within Base
```

That's not very good.